apps/finance/test/finance.js

```diff
@@ -382,9 +383,10 @@ contract('Finance App', accounts => {
 
         it('doesnt record payment for one time past transaction', async () => {
             await finance.newPayment(token1.address, recipient, 1, time, 1, 1, '')
-            return assertRevert(async () => {
-                await finance.getPayment(1)
```

Comment:
Debating whether or not we should keep these getters reverting if someone requests an out-of-bounds index

Reply:
Internal debate ended with adding checks for the length in all of the affected getters.

As we were using arrays before, and those throw `invalid opcode` when accessing out-of-bounds indices, we should aim for the same interface. I think it makes sense to for the caller to know if they've tried to access an out of bounds.